app/assets/stylesheets/adl-overrides/adl-overrides.scss

```diff
@@ -275,24 +275,33 @@ body.public-facing {
     background-color: $lightgrey;
   }
 
+  // Announcement Content Block
+
+  div#announcement {
+    color: $darkgrey;
+    font-size: 15px;
+    background-color: $white;
+    border: 2px solid $mediumgrey;
+    padding-top: 0;
+    border-radius: 0;
+    p {
+      margin-top: 10px;
+      font-weight: 600;
+    }
+}
+
   // ALERTS
-  .alert-info {
-    color: $black;
-    background-color: $lightgrey;
-    border-color: transparent;
-    border: 0 solid transparent;
-  }
-  .alert-success {
-    color: $white;
-    background-color: $darkyellow;
-    border-color: transparent;
-    border: 0 solid transparent;
-  }
-  .alert-warning {
-    color: $black;
-    background-color: $darkyellow;
-    border-color: transparent;
-    border: 0 solid transparent;
+
```

Comment:
To clarify, the three different alerts will now have the same color and background color.